errors.py

```diff
@@ -243,3 +242,13 @@ def __init__(self, errors, forceFail=False):
 		super(ReadFromClientError, self).__init__()
 		self.errors = errors
 		self.forceFail = forceFail
+
+
+class ViurException(Exception):
```

Comment:
> thats how it's handled anywhere else.

Obviously this exception is currently only used in the sendMail-method and for only that oversized. But the argument "we've always done it that way is" is quit weak. Of course it works with an default Exception or assertion. But IMO that's not much expressive like a Exception that indicates the error in the Exception/class-name and the details in the message.
Here I wanted to introduce the concept of own ViUR-Exception which indicates that a exception is explicit thrown by ViUR when the application-developer have done something wrong and not because there's a bug in ViUR.
There must be anywhere a begin, and now it's here because while I implemented this mail-update I looked for a suitable exception, found no one and created them.
Maybe it would be proper to introduce this not beside another feature, but in a new PR/Issue...